### DISCUSSION TITLE
Formatting in read mode?

### DISCUSSION THREAD

**Discussion by russellaugust** [chuangcaleb/obsidian-fountain-editor#23] [Ideas]
I know this plugin explicitly is called obsidian fountain editor, but it would be really nice if it also was optionally formatted in the read mode.

**Comment by chuangcaleb** [comment]
Yo! It would be really nice. How important would you triage this against the other tasks mentioned [here in MVP 2](https://github.com/users/chuangcaleb/projects/2/views/3)? Because personally, I hardly use Reading Mode anymore, so I don't have much personal incentive to work on this.

For now, it is possible to use the [Darakah/obsidian-fountain](https://github.com/Darakah/obsidian-fountain) plugin in parallel with this plugin for reading mode. I'm still waiting a reply on whether I can just copy over the code and supersede the 1.5year old plugin, but I think eventually it would make sense to add this feature in this plugin.

I did explain my game plan in the [Synergistic Tools](https://github.com/chuangcaleb/obsidian-fountain-editor/blob/master/docs/basic/synergistic-tools.md) docs about Reading Mode since you probably haven't read it yet, but I'll leave this thread open for anyone who might've missed it!

**Reply by russellaugust** [reply]
I'm going to explore that other fountain plugin more. I had lots of trouble with it in terms of formatting correctly.

**Reply by chuangcaleb** [reply]
Which plugin did you have formatting troubles with, my one or that other one? Because the other one is using a [time-tested npm package](https://www.npmjs.com/package/fountain-js) while I wrote my own parser, which is still half-baked haha

**Reply by russellaugust** [reply]
Well, that's very interesting. Yeah it's the other plug-in that hasn't been updated in a couple years. I found that it was just not formatting correctly in Obsidian and the page formatting looked off, and the margins were all wrong. It also felt like it was using the wrong font

**Reply by chuangcaleb** [reply]
Okay, hang on tight... this one might be a while. Meanwhile, do help keep a lookout for what can be improved. Showcasing stuff that you made can also help me identify areas for enhancement!
